Release r1.1 with 0.1.0-rc.1 (#15)
camaraproject/ModelAsAService: CHANGELOG.md

# Changelog ModelAsAService

## Table of contents

- **[r1.1](#r11)**

**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**

The below sections record the changes for each API version in each release as follows:

* for an alpha release, the delta with respect to the previous release
* for the first release-candidate, all changes since the last public release
* for subsequent release-candidate(s), only the delta to the previous release-candidate
* for a public release, the consolidated changes since the previous public release

<!--Repeat the below release section (header 1 and subsections) at the top of this file for each new (pre-)release-->

# r1.1

## Release Notes

This release contains the definition and documentation of
*  knowledge-base v0.1.0
*  qa-assistant-manage v0.1.0
*  qa-assistant-service v0.1.0

The API definition(s) are based on

> * Commonalities v0.5.0

Reviewer: ```suggestion
* Commonalities v0.6
```

> * Identity and Consent Management v0.3.0

Reviewer: ```suggestion
* Identity and Consent Management v0.4
```

## knowledge-base v0.1.0

> knowledge-base v0.1.0 is the first pre-release version of the knowledge-base API.

Reviewer: ```suggestion
knowledge-base v0.1.0-rc.1 is the first release candidate of version 0.1.0 of the API
```

- API definition **with inline documentation**:
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ModelAsAService/r1.1/code/API_definitions/knowledge-base.yaml&nocors)
  - [View it on Swagger Editor](https://camaraproject.github.io/swagger-ui/?url=https://raw.githubusercontent.com/camaraproject/ModelAsAService/r1.1/code/API_definitions/knowledge-base.yaml)
  - OpenAPI [YAML spec file](https://github.com/camaraproject/ModelAsAService/blob/r1.1/code/API_definitions/knowledge-base.yaml)

> 

Reviewer: ```suggestion

### Added
Update for Fall25 by @lihaojie-CMCC in [#11](https://github.com/camaraproject/ModelAsAService/pull/11),[13](https://github.com/camaraproject/ModelAsAService/pull/13)
Add API yaml files and API Test definitions by @lihaojie-CMCC in [#10](https://github.com/camaraproject/ModelAsAService/pull/10)

### Changed

### Fixed

### Removed
```

## qa-assistant-manage v0.1.0

> qa-assistant-manage v0.1.0 is the first pre-release version of the qa-assistant-manage API.

Reviewer: ```suggestion
qa-assistant-manage v0.1.0-rc.1 is the first release candidate of version 0.1.0 of the API
```

- API definition **with inline documentation**:
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/ModelAsAService/r1.1/code/API_definitions/qa-assistant-manage.yaml&nocors)
  - [View it on Swagger Editor](https://camaraproject.github.io/swagger-ui/?url=https://raw.githubusercontent.com/camaraproject/ModelAsAService/r1.1/code/API_definitions/qa-assistant-manage.yaml)
  - OpenAPI [YAML spec file](https://github.com/camaraproject/ModelAsAService/blob/r1.1/code/API_definitions/qa-assistant-manage.yaml)

> 

Reviewer: ```suggestion

### Added
Update for Fall25 by @lihaojie-CMCC in [#11](https://github.com/camaraproject/ModelAsAService/pull/11),[13](https://github.com/camaraproject/ModelAsAService/pull/13)
Add API yaml files and API Test definitions by @lihaojie-CMCC in [#10](https://github.com/camaraproject/ModelAsAService/pull/10)

### Changed

### Fixed

### Removed

```

## qa-assistant-service v0.1.0

> qa-assistant-service v0.1.0 is the first pre-release version of the qa-assistant-service API.

Reviewer: ```suggestion
qa-assistant-service v0.1.0-rc.1 is the first release candidate of version 0.1.0 of the API
```
